packages/esm-patient-chart-app/src/config-schema.ts

```diff
@@ -42,12 +42,43 @@ export const esmPatientChartSchema = {
     _description: 'Custom label to use on the start visit actions',
     _default: '',
   },
+  showServiceQueueFields: {
+    _type: Type.Boolean,
+    _description: 'Whether start visit form should display service queue fields`',
+    _default: false,
+  },
+  priorityConceptSetUuid: {
+    _type: Type.ConceptUuid,
+    _default: '9e123c90-76ac-4eaa-8d40-35577781eb46',
+  },
+  defaultPriorityConceptUuid: {
+    _type: Type.ConceptUuid,
+    _default: '9e123c90-76ac-4eaa-8d40-35577781eb46',
+  },
+  serviceConceptSetUuid: {
+    _type: Type.ConceptUuid,
+    _default: 'f9c6a3f8-a5d0-4034-ac77-e413c4b3c620',
+  },
+  statusConceptSetUuid: {
+    _type: Type.ConceptUuid,
+    _default: '13c9b9fb-3396-4139-98e1-59938d544168',
+  },
+  defaultStatusConceptUuid: {
+    _type: Type.ConceptUuid,
+    _default: '136203AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
```

Comment:
Adding a `description` for these config properties would be very useful.

Reply:
Trying to figure out what all this is... `showServiceQueueFields` has a self-referential description... what's it for, @denniskigen ? Are there designs somewhere? Sorry I'm going to need a lot of catching up around queues, I think